lib/components/QuizButton.dart

```diff
@@ -43,9 +38,13 @@ class _MyButtonState extends State<MyButton>{
                    color: const Color(0xFFffffff),
                    shape: new RoundedRectangleBorder(
                        borderRadius: const BorderRadius.all(const Radius.circular(16.0))),
-                   child: _displayText.endsWith(".png")? new Image.asset("$_displayText", width: 150.0, height: 150.0,) : new Text(_displayText,
-                       key: new Key("${widget.keys}"),
-                       style: new TextStyle(color: Colors.black, fontSize: ht * 0.05))
+                   child: widget.text.endsWith(".png")? new Image.asset("assets/$widget.text", width: wd * 0.4, height: ht * 0.12,) : new Container(
+                     width: wd * 0.4,
```

Comment:
Do not hard code the height and width. This button might be used in different ways. Let the component using this size it